app/templates/main/pagenation/src/index.js

```diff
@@ -113,11 +122,74 @@ const ResultList = () => {
 class ResultListContainer extends React.Component {
```

Comment:
I wrapped the actual Result List within another class Component (Hooks don't support Ref yet I think).. So all of the sorting functionalities happen below here.